### PR TITLE
#440 Scaffolding Improvements

### LINKED ISSUE #440: Add default HTML elements to scaffolding page
Example Elements:

1. Table
2. Lists
3. All form elements
4. Tables
5. abbr, pre, code, sub, sup
6. Blockquotes
7. Image left, right, center, none.

https://wp-themes.com/?p=36

### PR DESCRIPTION
**Pull request by mharis**
Closes #440

### DESCRIPTION ###

Added:

- Tables + Responsiveness.
- Link
- Ordered list
- Unordered list
- Blockquote
- Code tag + styling
- Em tag.
- Strong tag.
- Abbreviation tag.
- Acronym tag.
- Cite tag.
- Strikeout tag.
- Preformatted text and styling.
- Keyboard tag and styling
- Subscrip tag.
- Superscript tag.
- Variable tag.
- Address tag.
- Definition lists.
- Right aligned image.
- Left-aligned image.
- Centre-aligned image.
- Image with a caption.
- Youtube embed.
- Vimeo embed.
- Checkboxes.
- Radio boxes.
- Textarea.
- Numeric pagination.

## Table

gif

image

### OTHER ###
- [X] Is this issue accessible? (Section 508/WCAG 2.0AA)
- [X] Does this issue pass all the linting? (PHPCS, ESLint, SassLint)
- [X] Does this pass CBT? https://app.crossbrowsertesting.com/public/ieb67278514f6ec6/screenshots/z94765e359a0fbd8a0ef

### STEPS TO VERIFY ###

1. Create a scaffolding page locally.

### CODE REVIEW

File: Gulpfile.js [hunk 1]
```diff
@@ -29,6 +29,7 @@ const svgmin = require( 'gulp-svgmin' );
 const svgstore = require( 'gulp-svgstore' );
 const uglify = require( 'gulp-uglify' );
 const wpPot = require( 'gulp-wp-pot' );
+const replace = require( 'gulp-rename' );
```
Comment (coreymcollins) [inline Gulpfile.js]: What does this do/why was it added? This is the only question I have around the PR – everything else is looking good! Could just use some details on why this was required. I feel like I maybe remember a conversation you and I had (maybe?), but would be good to have it noted here as well.

**Review by coreymcollins — CHANGES_REQUESTED** [review]

**Comment by coreymcollins** [comment]
Can you also merge `master` down into your branch to resolve the various conflicts?

Thanks!

File: Gulpfile.js (hunk 1 above)
Comment (mharis) [reply Gulpfile.js]: https://webdevstudios.slack.com/archives/C162V538T/p1544212694089200?thread_ts=1544212070.085600&cid=C162V538T

Comment (coreymcollins) [reply Gulpfile.js]: Ahh, that's right. I knew I had seen it somewhere but couldn't recall. Thank you!

**Comment by coreymcollins** [comment]
In addition to merging `master` in, Jo had noted a typo in his original comment. `gulp-rename` should be `gulp-replace` instead.

**Review by coreymcollins — APPROVED** [review]

**Pull request merged**